src/js/lib/api/mock.js

```diff
@@ -1,13 +1,32 @@
 "use strict";
 
+//run this in terminal... USE_MOCK_API=true npm start
+//check index.js (this is why this command works)
+
+var userJSONObj={
+    	"user_name": "John Doe",
```

Comment:
Your editor should be configured with a linter to warn you about this, but we have our project set up so that we indent with 4 spaces, here it looks like you used tab characters. Some of the spacing here is also off.